Makefile

```diff
@@ -1,108 +1,128 @@
-# Note: This isn't really a build system
-# It's a bunch of command lines encoded
-# in a Makefile so I don't have to remember
-# multi part processes
-
-BBBUILDROOT=$(shell realpath ./bbbuildroot)
-M5BUILDROOT=$(shell realpath ./m5buildroot)
-OUTPUTS=$(shell realpath ./outputs/)
-BUILDROOT=$(BBBUILDROOT)/buildroot
-
-.PHONY: toolchain \
-	buildroot \
-	buildroot_m5 \
-	uboot \
-	uboot_m5 \
-	linux \
-	upload \
-	outputsdir \
-	kernel_m5.fit \
-	kernel_breadbee.fit \
-	rtk \
-	squeekyclean
+PWD = $(shell pwd)
+OUTPUTS=$(shell realpath $(PWD)/outputs/)
+LINUXROOT=$(shell realpath $(PWD)/linux)
+LINUX_CONFIG=$(LINUXROOT)/.config
+INUX_IMAGE=$(LINUXROOT)/arch/arm/boot/zImage
```

Comment:
Should this be LINUX_IMAGE?

Reply:
Ugh, yes. I thought I fixed this in my branch.